### PR TITLE
Backend skeleton

### PR DESCRIPTION
**Pull request by chelsea0x3b**

### CODE REVIEW

**Comment by chelsea0x3b** [comment]
[Create skeleton backend project (empty java project)](https://trello.com/c/EomSctMR/39-create-skeleton-backend-project-empty-java-project)

File: server/src/main/java/edu/jhu/server/App.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package edu.jhu.server;
+
+/**
+ * Hello world!
+ *
+ */
+public class App 
+{
+    public static void main( String[] args )
```
Comment (chelsea0x3b) [inline server/src/main/java/edu/jhu/server/App.java:9]: test comment

Comment (NebuchadnezzarXIII) [reply server/src/main/java/edu/jhu/server/App.java:9]: good skeleton, solid bones

**Review by NebuchadnezzarXIII — APPROVED** [review]
test approval

**Pull request merged**